tests/categorical_test.py

```diff
@@ -222,24 +223,24 @@ def testSaveAndLoadCategorical(self):
             self.assertTrue(dset_name in x)
             cat_from_hdf = x[dset_name]
 
-            expected_categories = ['string 1', 'string 2', 'string 3', 'string 4', 'string 5', 'string 6', 'string 7',
-                                   'string 8', 'string 9', 'string 10']
+            expected_categories = [f"string {i}" for i in range(1, num_elems)]
 
             # Note assertCountEqual asserts a and b have the same elements in the same amount regardless of order
             self.assertCountEqual(cat_from_hdf.categories.to_ndarray().tolist(), expected_categories)
 
             # Asserting the optional components and sizes are correct for both constructors should be sufficient
             self.assertTrue(cat_from_hdf.segments is not None)
             self.assertTrue(cat_from_hdf.permutation is not None)
-            self.assertTrue(cat_from_hdf.size is 10)
+            print(f"==> cat_from_hdf.size:{cat_from_hdf.size}")
+            self.assertTrue(cat_from_hdf.size == num_elems-1)
 
     def testSaveAndLoadCategoricalMulti(self):
         """
         Test to build a pseudo dataframe with multiple categoricals, pdarrays, strings objects and successfully
         write/read it from HDF5
         """
-        c1 = ak.Categorical(ak.array(['abc', 'def', 'ghi', 'jkl']))
-        c2 = ak.Categorical(ak.array(['mno', 'pqr', 'stu', 'vwx', 'yz']))
+        c1 = self._getCategorical(prefix="c1", size=51)
+        c2 = self._getCategorical(prefix="c2", size=52)
         pda1 = ak.zeros(5)
         strings1 = ak.random_strings_uniform(9, 10, 5)
```

Comment:
This test fails for me at 6+ locales now instead of 5+. Guessing it's because `pda1`/`strings1` are of length 5. Bumping those to 52 (no idea if that makes sense for a size) seems to work at 16 nodes.

Reply:
Yup, you are correct, I forgot to bump all of the things that go in the map; tunnel vision doh!

Reply:
Ok, sorry for the delay, I pushed the additional changes to increase the number of elements in those two arrays.